tests/Scripts/have-sanitizer

```diff
@@ -6,4 +6,4 @@ test "${USING_BUILD_DIRECTORY}" = 1 || exit 1
 
 base=$(cd $(dirname $0) && pwd)
 cache=$(${base}/build-directory)/CMakeCache.txt
-test -e ${cache} && grep -q "HILTI_HAVE_SANITIZER.*=yes" ${cache}
+test -e ${cache} && grep -q "USE_SANITIZER.*=.*" ${cache}
```

Comment:
Should this just be `USE_SANITIZERS` instead of the wildcard? I don't see it defined any other way.

Reply:
Thanks for the careful review. The variable is indeed called `USE_SANITIZERS`, but I needed to match `USE_SANITIZER:STRING=...` here. I now use a more narrow pattern.